data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -51,9 +51,9 @@
             {EI_BIGMAP}
         [/part]
         [part]
-            # The River Guard posts had been built in 470 YW. They were
-            # abandoned in 544 YW; the wave of colonization had begun around
-            # 530 YW.
+            #po: The River Guard posts had been built in 470 YW. They were
+            #po: abandoned in 544 YW; the wave of colonization had begun around
+            #po: 530 YW. It is now 625 YW.
```

Comment:
This intro starts in 625 YW; the king's forces arrive at the outposts in 626 YW.